header.html

```diff
@@ -13,26 +13,26 @@
 </head>
 <body>
     <header class="header">
-        <img src="img/header-shapesmobile@2x.svg" alt="" class="bg-shapes">
-            <nav>
-                <ul class="navigation">
-                    <li class="logo">
-                        <a href="#">My Logo</a>
-                    </li>
+            <nav class="navigation">
+                <p class="logo">
+                    <a href="#">My Logo</a>
+                </p>
```

Comment:
- [ ] Kindly note that you are required to stick as much as possible to the provided Figma design in fonts, colors, etc. The font size for this logo should be 18px as specified in the Figma design.